src/routes/api/commits-from-date/reset-failed-and-pending-deployment-cursors.ts

```diff
@@ -29,10 +29,7 @@ export const ResetFailedAndPendingDeploymentCursorPost = async (req: Request, re
 			where: {
 				[Op.and]: {
 					"deploymentStatus": {
-						[Op.or]: {
-							[Op.is]: undefined,
-							[Op.in]: ["pending", "failed"]
-						}
+						[Op.in]: ["pending", "failed"]
```

Comment:
curious why this was changed?

Reply:
There're 4 millions rows of `null` lol
But when I checked the Db, they are not needed to be reset
```
=> select count(*) from "RepoSyncStates" where ( "deploymentStatus" is null and "deploymentCursor" is not null);
 count
-------
     0
(1 row)
```

I can however to fine tune above sql to include `status` is null and `cursor` is not null, but wonder will that make above code too complicated? Thoughts?